src/vidible-player.js

```diff
@@ -179,7 +211,11 @@
                                 },
                                 isMuted: function () {
                                     try {
-                                        return vidiblePlayer.getPlayerInfo().volume === 0;
+                                        var volume = vidiblePlayer.getPlayerInfo().volume;
+                                        if(volume === undefined || volume == null)
+                                            return undefined;
+                                        else
+                                            return vidiblePlayer.getPlayerInfo().volume === 0;
                                     } catch (e) {
```

Comment:
Is there a situation where vidiblePlayer.getPlayerInfo().volume throws undefined?
If yes we should check using if statements.